pkg/apply/mutator/apply_time_mutator.go

```diff
@@ -134,25 +136,26 @@ func (atm *ApplyTimeMutator) Mutate(ctx context.Context, obj *unstructured.Unstr
 		klog.V(5).Infof("substitution: targetRef=(%s), sourceRef=(%s): sourceValue=(%v), token=(%s), oldTargetValue=(%v), newTargetValue=(%v)",
 			targetRef, sourceRef, sourceValue, sub.Token, targetValue, newValue)
 
-		// update target field in target resource
+		// update target field in target object
 		err = writeFieldValue(obj, sub.TargetPath, newValue)
 		if err != nil {
-			return mutated, reason, fmt.Errorf("failed to set field in target resource (%s): %w", targetRef, err)
+			return mutated, reason, fmt.Errorf("failed to set field in target object (%s): %w", targetRef, err)
 		}
 
 		mutated = true
-		reason = fmt.Sprintf("resource contained annotation: %s", mutation.Annotation)
+		reason = fmt.Sprintf("object contained annotation: %s", mutation.Annotation)
 	}
 
 	if mutated {
-		klog.V(4).Infof("mutated target resource (%s):\n%s", targetRef, object.YamlStringer{O: obj})
+		klog.V(4).Infof("mutated target object: %s", targetRef)
+		klog.V(7).Infof("mutated target object YAML:\n%s", object.YamlStringer{O: obj})
```

Comment:
I'm curious why logging the whole object yaml instead of the fields being mutated.

Reply:
It's mostly just for debugging when the JSONPath fails to identify the field you wanted.

Reply:
Ack